examples/lm1b/train.py

```diff
@@ -293,30 +365,44 @@ def main(argv):
 
   rng = random.PRNGKey(random_seed)
```

Comment:
correct - because of the simple dynamic batching implementation this is forced to be single-host example, but this is good to add anyway as a clear example of that use.